public/stylesheets/style.css

```diff
@@ -64,23 +64,38 @@ footer h5{
     width: 550px;
 }
 
-@media screen and (max-width: 570px) {
```

Comment:
Let's do this by creating a class we attach to the paragraph with a fixed height. That way if the text changes we can just adjust that one value instead of having to compute new values for each individual box.